Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGToDBAnalyzer.cs

```diff
@@ -83,79 +162,258 @@ public override void Prepare()
             }
             catch (Exception e)
             {
-                Console.WriteLine("No such dir or could not delete dir with exception {0}.\nIf dir still exists, this analyzer will append data to existing DB.", e);
+                Console.WriteLine("Directory entered does not exist, creating directory for DB.", e);

```

Comment:
>Console.WriteLine("Directory entered does not exist, creating directory for DB.", e); [](start = 16, length = 85)

how about checking for directory existence first, instead of catching an exception? Moreover, Delete might explode for many reasons, so the message might not be accurate

Reply:
Yes, this was in a todo of mine. Was gonna send it in a different PR, but will do that now instead.

---
In reply to: [308905562](https://github.com/microsoft/BuildXL/pull/671#discussion_r308905562) [](ancestors = 308905562)